src/Surfnet/ServiceProviderDashboard/Application/Command/Service/CreateServiceCommand.php

```diff
@@ -30,160 +32,112 @@ class CreateServiceCommand implements Command
 {
     /**
      * @var string
-     * @Assert\Uuid(strict=false)
      */
+    #[Assert\Uuid(strict: false)]
     private $guid;
 
     /**
-     * @var string
-     * @Assert\NotBlank
-     * @SpDashboardAssert\UniqueTeamName()
+     * @var                                string
      */
+    #[SpDashboardAssert\UniqueTeamName()]
+    #[Assert\NotBlank]
     private $teamName;
 
     /**
      * @var string
-     * @Assert\NotBlank
      */
+    #[Assert\NotBlank]
     private $name;
 
-    /**
-     * @var string
-     * @Assert\NotBlank
-     * @Assert\Email()
-     */
-    private $teamManagerEmail;
+    #[Assert\NotBlank]
+    #[Assert\Email]
+    private ?string $teamManagerEmail = null;
 
     /**
      * @var string
      */
     private $institutionId;
 
-    /**
-     * @var bool
-     */
-    private $productionEntitiesEnabled = false;
+    private bool $productionEntitiesEnabled = false;
 
-    /**
-     * @var bool
-     */
-    private $privacyQuestionsEnabled = true;
+    private bool $privacyQuestionsEnabled = true;
 
-    /**
-     * @var bool
-     */
-    private $clientCredentialClientsEnabled = false;
+    private bool $clientCredentialClientsEnabled = false;
 
-    /**
-     * @var string
-     * @Assert\NotBlank
-     */
-    private $serviceType = Service::SERVICE_TYPE_NON_INSTITUTE;
+    #[Assert\NotBlank]
+    private string $serviceType = Service::SERVICE_TYPE_NON_INSTITUTE;
 
-    /**
-     * @var string
-     * @Assert\NotBlank
-     */
-    private $intakeStatus = Service::INTAKE_STATUS_NO;
+    #[Assert\NotBlank]
+    private string $intakeStatus = Service::INTAKE_STATUS_NO;
 
-    /**
-     * @var string
-     */
-    private $contractSigned = Service::CONTRACT_SIGNED_NO;
+    private string $contractSigned = Service::CONTRACT_SIGNED_NO;
 
-    /**
-     * @var string
-     */
-    private $surfconextRepresentativeApproved = Service::SURFCONEXT_APPROVED_NO;
+    private string $surfconextRepresentativeApproved = Service::SURFCONEXT_APPROVED_NO;
 
-    /**
-     * @var string
-     * @Assert\NotBlank
-     */
-    private $organizationNameNl;
+    #[Assert\NotBlank]
+    private ?string $organizationNameNl = null;
 
-    /**
-     * @var string
-     * @Assert\NotBlank
-     */
-    private $organizationNameEn;
+    #[Assert\NotBlank]
+    private ?string $organizationNameEn = null;
 
     /**
      * @param string $guid
      */
```

Comment:
Missing type definition for method, redundant phpdoc.

Reply:
unfortunatly, the typing is not consistent enough to trust the phpdoc to apply it to the vars. Therefor, a lot of vars arent' still typed.

Reply:
I see your dilemma, and making the app type-safe in this upgrade is not priority #1. So feel free to leave as-is.
However, a lot of redundant PHPDoc can still be removed.